eng/pipelines/templates/steps/install-dotnet.yml

```diff
@@ -17,19 +17,18 @@ steps:
     inputs:
       useGlobalJson: true
       performMultiLevelLookup: true
-  - task: UseDotNet@2 # We need .NET 6 for azure-sdk-tools and the .NET 6 tests
-    condition: and(succeeded(), ne(variables['Agent.OS'], 'Windows_NT')) # Windows supports MultiLevelLookup and doesn't need explicit framework installation
```

Comment:
https://learn.microsoft.com/en-us/dotnet/core/compatibility/deployment/7.0/multilevel-lookup this doesn't work anymore for applications targeting .NET7+